epochs/hippieeg/config/snakebids.yml

```diff
@@ -36,8 +36,7 @@ pybids_inputs:
       - task
       - run
 
-pybids_inputs_annot:
-  bold:
+  annotations:
```

Comment:
Basically the only change that you did was creating one unique input dict with multiple keys, right?